package.json

```diff
@@ -40,5 +40,10 @@
     "sdk/route",
     "sdk/examples",
     "cli"
-  ]
+  ],
+  "overrides": {
+    "ts-jest": {
+      "@types/jest": "^29.5.12"
```

Comment:
Was getting `Missing: @types/jest@27.5.2 from lock file` without this. The wormchain SDK has ^27 as a peer dep. Didn't happen to me locally, and the lockfile was indeed updated, confusing.